src/components/cardbuilder/card.css

```diff
@@ -280,7 +281,7 @@ button {
 
 .visualCardBox {
     box-shadow: 0 0.0725em 0.29em 0 rgba(0, 0, 0, 0.37);
-    border-radius: .145em;
+    border-radius: 15px
```

Comment:
`em` is relative to the size of `m`, `px` is number of pixels.
This will likely look wildly different on a high DPI screen compared to a normal computer screen.